src/sigal/gallery.py

```diff
@@ -935,8 +936,23 @@ def remove_files(self, medias):
 
     def process_dir(self, album, force=False):
         """Process a list of images in a directory."""
+        def forcing(a):
```

Comment:
Please add tests for this function (so it should be importable, you can move it to `sigal.utils`).
Also I think it should have a better name, maybe `should_reprocess_album` ?

Reply:
How do you feel about album.is_forcing()?